mmv1/products/spanner/InstanceConfig.yaml

```diff
@@ -14,20 +14,106 @@
 --- !ruby/object:Api::Resource
 name: 'InstanceConfig'
 base_url: 'projects/{{project}}/instanceConfigs'
+update_verb: :PATCH
 description: |
   A possible configuration for a Cloud Spanner instance. Configurations
   define the geographic placement of nodes and their replication.
-readonly: true
-# Used as a resource ref
-exclude: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/spanner/'
+  api: 'https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instanceConfigs'
+async: !ruby/object:Api::OpAsync
```

Comment:
Most of the things in this async block are unused; see https://github.com/hashicorp/terraform-provider-google/issues/17098 for details. I'd recommend removing them for clarity but it's not strictly required.